### PR TITLE
Update README for Run MATLAB Command

### PR DESCRIPTION
**Pull request by mw-hrastega**
Updated the text and examples. When reviewing the examples, please pay attention to the text in the YAML scripts.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,53 +1,74 @@
...
+        uses: matlab-actions/setup-matlab@v0
+      - name: Run commands
         uses: matlab-actions/run-command@v0
         with:
-            command: disp('hello world')
+          command: 'results = runtests, assertSuccess(results);'
 ```
 
+## Run MATLAB Command
+When you define your workflow in the `.github/workflows` directory of your repositoy, you can specify the **Run MATLAB Command** action using the `run-command` key. The action requires an input.
```
Comment (jwpereira) [inline README.md:55]:
```suggestion
When you define your workflow in the `.github/workflows` directory of your repository, you can specify the **Run MATLAB Command** action using the `run-command` key. The action requires an input.
```

Comment (jwpereira) [inline README.md:55]: In practice, I don't think users will be able to call `run-command` without qualifying it with our org and a version 🤔 
They'd have to use `matlab-actions/run-command@v0` 🤔

**Review by jwpereira — APPROVED** [review]
Now this.. is a real doc 😁 
Thanks, @mw-hrastega! 😄

File: README.md (hunk 1 above)
Comment (mw-hrastega) [reply README.md:55]: Fixed.

Comment (mw-hrastega) [reply README.md]: How about the updated version?

File: README.md [hunk 2]
```diff
@@ -1,53 +1,74 @@
...
+The [Run MATLAB Command](#run-matlab-command) GitHub&reg; action enables you to execute a MATLAB&reg; script, function, or statement on a [GitHub-hosted](https://docs.github.com/en/free-pro-team@latest/actions/reference/specifications-for-github-hosted-runners) or [self-hosted](https://docs.github.com/en/free-pro-team@latest/actions/hosting-your-own-runners/about-self-hosted-runners) runner:
 
-MATLAB exits with exit code 0 if the specified script, function, or statement
-executes successfully without error. Otherwise, MATLAB terminates with a nonzero
-exit code, which causes the build to fail. You can use the
-[`assert`](https://www.mathworks.com/help/matlab/ref/assert.html) or
-[`error`](https://www.mathworks.com/help/matlab/ref/assert.html) functions in
-the command to ensure that builds fail when necessary. When you use this task,
-all of the required files must be on the MATLAB search path.
+- If you want to use a self-hosted runner, you must set up a computer with MATLAB (R2013b or later) as your runner. The action uses the first MATLAB version on the runner's path.
```
Comment (mcafaro) [inline README.md]: "The action uses the first MATLAB version on the runner's **system** path."

File: README.md [hunk 3]
```diff
@@ -1,53 +1,74 @@
...
+  my-job:
+    name: Run MATLAB Script
+    runs-on: self-hosted
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+      - name: Run script
+        uses: matlab-actions/run-command@v0
+        with:
+          command: 'myscript'
```
Comment (mcafaro) [inline README.md:28]: We may want to consider not showing the command wrapped in single quotes. I believe these quotes are [generally unneeded](https://stackoverflow.com/questions/19109912/yaml-do-i-need-quotes-for-strings-in-yaml) and a user may want to use single quotes in their actual MATLAB command, in which case they would need to learn how to properly escape them.

File: README.md [hunk 4]
```diff
@@ -1,53 +1,74 @@
...
-      - name: Run "disp('hello world')" directly using MATLAB
+      - name: Check out repository
+        uses: actions/checkout@v2
+      - name: Install MATLAB
+        uses: matlab-actions/setup-matlab@v0
+      - name: Run commands
         uses: matlab-actions/run-command@v0
         with:
-            command: disp('hello world')
+          command: 'results = runtests, assertSuccess(results);'
```
Comment (mcafaro) [inline README.md:51]: Again, perhaps lose the quotes.

File: README.md [hunk 5]
```diff
@@ -1,53 +1,74 @@
...
 ```
 
+## Run MATLAB Command
+When you define your workflow in the `.github/workflows` directory of your repository, you can specify the **Run MATLAB Command** action as `matlab-actions/run-command@v0`. The action requires an input.
+
+Input                     | Description    
+------------------------- | --------------- 
+`command`                 | (Required) Script, function, or statement to execute. If the value of `command` is the name of a MATLAB script or function, do not specify the file extension. If you specify more than one MATLAB command, use a comma or semicolon to separate the commands.<br/>**Example:** `'myscript'`<br/>**Example:** `'results = runtests, assertSuccess(results);'`
```
Comment (mcafaro) [inline README.md:59]: Quotes...

**Review by mcafaro — APPROVED** [review]
Minor suggestions

File: README.md (hunk 2 above)
Comment (mw-hrastega) [reply README.md]: Done.

File: README.md (hunk 4 above)
Comment (mw-hrastega) [reply README.md]: Makes sense.

File: README.md (hunk 3 above)
Comment (mw-hrastega) [reply README.md]: Good point. Thanks.

**Pull request merged**